src/pybind/mgr/tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
     py3,
-    py37,
+    py39,
```

Comment:
ditto. justifications needed. currently we build packages for CentOS 8, openSUSE 15.3 and Ubuntu focal, and test the built packages on CentOS 8 and Ubuntu focal. the tox based test is the first guard for the suspicious changes. it is supposed to test the typical settings and fail the tests which would otherwise break the tests in our integration tests exercising the packages. 

probably we should bump the version to 3.9 in near future. but probably not now.